projects/fsastorefrontstyles/scss/base/building-blocks.scss

```diff
@@ -253,3 +253,43 @@
 .z-index-1 {
   z-index: 1;
 }
+
+@keyframes slide-in {
+  0% {
+    transform: translateX(100%);
+    -webkit-transform: translateX(100%);
```

Comment:
Since we no longer support older web browsers `webkit-transform` is not needed any more.

Reply:
`-webkit-transform` still here 😃